packages/eslint-plugin/configs/test-unit.js

```diff
@@ -1,3 +1,9 @@
 module.exports = {
 	extends: [ 'plugin:jest/recommended' ],
+	rules: {
+		'jest/expect-expect': [
+			'error',
+			{ assertFunctionNames: [ 'expect', 'measurePerformance' ] },
+		],
+	},
```

Comment:
As referenced in the [Reassure docs](https://callstack.github.io/reassure/docs/installation/#optional-eslint-setup), this has the linter consider `measurePerformance` an assertion. 

It is possible there may be a better location to place this new configuration, but the `test-unit` file did seem appropriate.

Reply:
Ah, good call. I agree this is better than forcing the other assertions.